resources/public/api/conf/1.0/application.yaml

```diff
@@ -435,6 +674,40 @@ components:
       description: >-
         Identity of the trader as known to HMRC. In the UK this is the 14-17
         character EORI number.
+    updatedSinceParam:
+      in: query
+      name: updatedSince
+      description: >-
+        Date value, all records after which have to be retrieved from the
+        backend; This should use standard ISO8601 format (yyyy-mm-dd hh:mm:ssZ).
+      schema:
+        type: string
+        format: date-time
+        pattern: >-
+          ^([1-9][0-9]{3})-(0[0-9]|1[0-2])-(0[1-9]|[12][0-9]|3[01])[T]([0-1][0-9]|2[0-3]):([0-5][0-9]):([0-5][0-9])Z$
```

Comment:
should be this (more readable):
pattern: ^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}Z$